lec15-git-projects.Rmd

```diff
@@ -477,6 +478,233 @@ The 'GitHub flow':
 7. Once your edits have been merged, delete your new edits-containing branch
 8. Sync your fork to be up to date with the new changes
 
+## Collaborating with GitHub without the command line
+
+The following section is a modified version of the above section that covers
+collaboration with GitHub **without** the use of the command line.
+
+All groups have been provided with existing repositories, but a new repository
+can be made by clicking on the `+` in the top right of the page and selecting
+'New repository'. For now, however, navigate to your provided group repo.
+
+### Creating a fork
+
+The repos that have already been created can be thought of as 'main repos',
+which will contain the 'primary' version of the repo at any given point in
+time. Although in this version of the lesson, we *will* in fact work within
+this 'main repo' we will still begin by _forking_ the repo. When a given user
+forks a repo, GitHub creates a user-specific copy of the repo and all its
+files.
+
+To fork your group's repo, navigate to the repo's page and click on the 'Fork'
+button at the top right of the page. Following a brief load screen, GitHub will
+redirect you to your new, forked repo.
+
+You'll notice that on the top left of this repo page, the repo's name will be
+'[your username] / [repo name]', as opposed to 'EEB313 / [repo name]'.
+Furthermore, GitHub will indicate that this is a fork right underneath said
+repo name ('forked from eeb313-[year]/ [repo name]'). 
+
+For the duration of this lesson, we will stick to the `eeb313` version of your
+group's repo. Although the fork-based workflow (see above) is more common in
```

Comment:
Are you sure the fork based approach requires the command line? Seems like you should be able to edit files on your fork and submit PRs to the master repo all through the browser. Don't feel the need to change this. Just seems strange that that functionality wouldn't exist yet.

Reply:
My understanding was that edits/PRs can all be done without the command line, but that syncing a fork (ie `git pull`) couldn't be done via GitHub. That being said, apparently there's a [hacky way to do that](https://www.sitepoint.com/quick-tip-sync-your-fork-with-the-original-without-the-cli/) by creating a PR from the fork and switching the bases... so I'm going to reword this and say we're just sticking to the branch workflow for simplicity's sake.